src/utils/index.ts

```diff
@@ -91,3 +103,18 @@ export function getOptionValue(options: any, short: string, long: string): any {
         return options[long]
     }
 }
+
+export function convertSnakeToPascal(s: string): string {
+    return convertSnakeToCamel(s).replace(/^\w/, c => c.toUpperCase())
+}
+
+export function inferTypeNameFromArrayName(arrayName: any): string {
+    // If the last character of the array name is 's', it removes the last character and conver remaining string to Pascal Case,
+    // Otherwise, it simply converts the name to Pascal Case. 
+    // It sometime generates name that may not be appropriate e.g.vendorReturnPolicies: [AAA_VendorReturnPolicie] 
```

Comment:
Is this an OK tradeoff?

Reply:
I retained the array check because i saw instances where objects were named in plural. e.g.
```type AAA_Images {
  # A list of alternate images
  alternateImages: [String]
  # Base URL to reference an image
  baseUrl: String
  # Name of the primary image
  primaryImage: String
  # Name of the swatch image
  swatchImage: String
}```